src/main/java/com/google/firebase/ImplFirebaseTrampolines.java

```diff
@@ -61,4 +68,12 @@ public static String getPersistenceKey(String name, FirebaseOptions options) {
     app.addService(service);
     return service;
   }
+
+  public static ThreadFactory getThreadFactory(@NonNull FirebaseApp app) {
+    return app.getThreadFactory();
+  }
+
+  public static <T> Task<T> submit(@NonNull FirebaseApp app, @NonNull Callable<T> command) {
```

Comment:
submitCallable maybe? 'submit' seems generic in the context of this class.

Reply:
Done